modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -43,6 +43,14 @@
 	}
 }
 
+.ddm-form-builder-wrapper {
+	button[disabled],
+	input[disabled],
+	textarea[disabled] {
+		pointer-events: none;
+	}
+}
+
```

Comment:
@alinedoleron, will this CSS allow the `cursor-pointer` to be `not-allowed` when the user hovers over the field **Field ID**? Shouldn't we do something like [this](https://github.com/natocesarrego/liferay-portal/pull/657/commits/5ade9149a2c54429388bae7baebf745d801936ed)? Can you please clarify?

Reply:
Hi @natocesarrego. This case is a little bit different from field options creation. When we set the fieldID to be readOnly the input field is being created by Clay. So there's no **disabled class**, but there is an attribute called **disabled**  instead. This attribute has the property **cursor: not-allowed** which was overwritten by some change of css that disabled pointer events. This behavior should be made only for Form Builder fields, so I added this class to specify and not affect the sidebar fields.

Please, check the difference in the code below:

Field ID:
`<input aria-label="text" disabled name="_com_liferay_dynamic_data_mapping_form_web_portlet_DDMFormAdminPortlet_ddm$$name$Bx2cJBnj$0$$en_US" placeholder="" type="text" class="form-control ddm-field-text" value="SelectFromList">`

FieldID of options:
`<input class="disabled key-value-input" readonly="" type="text" value="Option">`

Reply:
Great! Thanks for the clarification, @alinedoleron. 🙂